HelpSource/Tutorials/Getting-Started/04-Functions-and-Other-Functionality.schelp

```diff
@@ -3,7 +3,9 @@ summary:: Getting Started With SuperCollider
 categories:: Tutorials>Getting-Started
 related:: Tutorials/Getting-Started/00-Getting-Started-With-SC
 
-The easiest way to get sound from SC is to use a Function. Below is a simple example of this. Execute this (after making sure the server is booted), and when you're sick of it, press Cmd - . (that's hold down the command key and press the period or fullstop key) to stop the sound. This will always stop all sound in SC. You'll be using it a lot, so commit it to memory.
+Warning! If you are using headphones or external speakers, it is recommended to set your volume low before playing the following example and then bring it up to a comfortable level. Raw synthesized tones can sometimes seem louder than music and the other forms of audio we usually listen to.
+
+The easiest way to get sound from SC is to use a Function. Below is a simple example of this. Execute this (after making sure the server is booted), and when you're sick of it, press Cmd-. (i.e hold down the command key and press the period or fullstop key) to stop the sound. This will always stop all sound in SC. You'll be using it a lot, so commit it to memory.
```

Comment:
thanks!
here is an alternative suggestion:

The easiest way to get sound from SC is to play a Function. Below is a simple example of this. Run this (after making sure the server is booted), and when you've got enough, press Cmd-. (i.e hold down the command key and press the period or fullstop key) to stop the sound. This will always stop all running processes in SC. You'll be using it a lot, so learn it by heart.

Reply:
also, can probably remove the part about how to press `Cmd-.` (or change it to `Ctrl-. (Cmd-. on macOS)`) since that is now explained at length in the prologue

Reply:
Good call! Updating with pieces of both of your suggestions